packages/reports/src/Containers/open-positions.tsx

```diff
@@ -469,9 +473,9 @@ const OpenPositions = ({
     const [has_multiplier_contract, setHasMultiplierContract] = React.useState(false);
     const previous_active_positions = usePrevious(active_positions);
     const contract_types = [
-        { text: localize('Options'), is_default: !is_multiplier && !is_accumulator },
-        { text: localize('Multipliers'), is_default: is_multiplier },
-        { text: localize('Accumulators'), is_default: is_accumulator },
+        { text: localize('Options'), value: 'Options', is_default: !is_multiplier && !is_accumulator },
```

Comment:
Since the value is not something to be shown to clients, can we make it lowercase e.g `options`?

Reply:
yes